.env.demo

```
@@ -1,191 +1,414 @@
-LOG_LEVEL=debug # ELK log level
-ELK_LOG_PATH= "http://localhost:9200/" # ELK log path
-ELK_USERNAME=elastic # ELK user username
-ELK_PASSWORD=xxxxxx # ELK user password
+# ELK flag
+ELK_LOG=false
+
+# ELK log level
+LOG_LEVEL=debug
+
+# ELK log path
+ELK_LOG_PATH= "http://localhost:9200/"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)